tensorflow/python/autograph/converters/logical_expressions.py

```diff
@@ -50,8 +51,11 @@ def _overload_of(self, operator):
     if op_type in LOGICAL_OPERATORS:
       return LOGICAL_OPERATORS[op_type]
     if self.ctx.user.options.uses(converter.Feature.EQUALITY_OPERATORS):
-      if op_type in EQUALITY_OPERATORS:
-        return EQUALITY_OPERATORS[op_type]
+      deprecation.deprecated(None,
+        ("tf.autograph.experimental.Feature.EQUALITY_OPERATORS "
+        "is deprecated and it is alredy enabled by default"))
+    if op_type in EQUALITY_OPERATORS:
```

Comment:
Let's remove this one, I think it's what causes the failures. Let's remove the EQUALITY_OPERATORS dict altogether, and let Tensor's `__eq__` handle things.